apps/site/src/components/common/NFTCard/index.tsx

```diff
@@ -1,31 +1,35 @@
 import classNames from 'classnames';
 import useTranslation from 'next-translate/useTranslation';
-import Image from 'next/image';
+import Image from 'next/future/image';
 import { Button, Chip } from '..';
 
 interface Props {
   big?: boolean;
+  blurhash?: string;
   chain: string;
   className?: string;
   contract: string;
   creatorName: string;
   creatorAvatar?: string;
   image_url: string;
   name: string;
+  unoptimized?: boolean;
   token_id: string;
   url: string;
   variant?: 'featured' | 'view';
 }
 
 export default function NFTCard({
   big = false,
+  blurhash,
   chain,
   className,
   contract,
   creatorName,
   creatorAvatar,
   image_url,
   name,
+  unoptimized = false,
```

Comment:
```suggestion
  unoptimized,
```

so that we follow the default value of `next/future/image`'s default value for `unoptimized`.